### ISSUE TITLE
How does the batchsize and lr influence the performance

### ISSUE THREAD

**Issue by xyimaging** [cwmok/C2FViT#6]
First of all, thanks so much for your good work!

I'm training on some 256x256x256 images and I find there is still a lot of free memory, so I'd like to use larger batchsize and lr to speed up the training. It would be great if you have any ablation study or insight on these 2 factors' influence. Thanks!

**Comment by cwmok** [comment]
Hi @xyimaging,

I didn't try using a larger batch size/lr. Empirically, I found that the current learning rate work well for many CNN-based registration networks. 

Besides, the `scale_factor` of the input image also greatly affects the registration accuracy. Feel free to let me know if you have any interesting results. :D

**Comment by xyimaging** [comment]
Thanks a lot for your sharing!
